x-pack/plugins/fleet/server/services/security/uninstall_token_service/index.ts

```diff
@@ -0,0 +1,412 @@
+/*
```

Comment:
Got curious, why is this service inside a folder and not next to the others? I'm not opposed to that but just wondering if there is a specific reason for that.

Reply:
just felt like it was getting a bit crowded